From 0b6eef3f1e971a93aaba5f99c8b289a787f6e67a Mon Sep 17 00:00:00 2001
From: Antonio Terceiro <terceiro@debian.org>
Date: Sat, 29 Mar 2014 12:07:39 -0300
Subject: [PATCH 2/2] tests/adt-run: remove unecessary imports

These actually make the package build fail because pyflakes fails on
them.
---
 tests/adt-run | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tests/adt-run b/tests/adt-run
index cac516e..8f6dc31 100755
--- a/tests/adt-run
+++ b/tests/adt-run
@@ -31,8 +31,6 @@ import unittest
 import tempfile
 import shutil
 import fnmatch
-import fcntl
-import time
 
 test_dir = os.path.dirname(os.path.abspath(__file__))
 root_dir = os.path.dirname(test_dir)
-- 
1.9.1

